### PR TITLE
Containerd v1.2.2

### PR DESCRIPTION
**Pull request by ijc**
Picks up v.1.2.1 through to 1.2.2.
https://github.com/containerd/containerd/releases/tag/v1.2.1
https://github.com/containerd/containerd/releases/tag/v1.2.2

Slight tweak needed to our rebuild of the wireguard userspace since the upstream/edge apk we rebuild gained another file at some point.

### CODE REVIEW

File: pkg/runc/Dockerfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ RUN \
   make \
   && true
 ENV GOPATH=/go PATH=$PATH:/go/bin
-ENV RUNC_COMMIT=58592df56734acf62e574865fe40b9e53e967910
```
Comment (rn) [inline pkg/runc/Dockerfile]: shouldn't this also bump the `linuxkit/alpine` hash?

Comment (ijc) [reply pkg/runc/Dockerfile]: I'm  a couple of mins away from pushing the bump, so we may as well wait for the CI again.

Comment (rn) [reply pkg/runc/Dockerfile]: did you also rebuild the `runc` package and bumped the hash? I don't trust the GH ordering of commits...

Comment (ijc) [reply pkg/runc/Dockerfile]: Yes, I bumped the hash in the same commit (the only new one, which is now the head of the branch) as I changed the `FROM` and I rebuilt manually on all three arches.

**Pull request merged**